flowclient/flowclient/aggregates.py

```diff
@@ -707,6 +708,8 @@ def flows_spec(
         Query which maps individuals to single location for the "origin" period of interest.
     to_location: dict
         Query which maps individuals to single location for the "destination" period of interest.
+    join_type: str default "inner"
+        Join type to use to build the flows
```

Comment:
You'll need to also add this param description to the docstring of flowclient.aggregates.flows (the function that returns an API query object) - the arguments are automatically picked up from flows_spec, but unfortunately the docstring still needs to be manually duplicated.